mimic/model/keypair_objects.py

```diff
@@ -0,0 +1,100 @@
+"""
+Keypair objects for mimic
+"""
+
+import json
+
+from characteristic import attributes, Attribute
+
+
+@attributes(['name', 'public_key'])
+class KeyPair(object):
+    """
+    A KeyPair object
+    """
+    fingerprint = "aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa:aa"
+
+    def key_json(self):
+        """
+        Serialize a :obj:`Keypair` into JSON
+        """
+        return {
+            "keypair": {
+                "name": self.name,
+                "public_key": self.public_key,
+                "fingerprint": self.fingerprint
+            }
+        }
+
+
+@attributes(
+    ["tenant_id", "region_name", "clock",
+     Attribute("keypairs", default_factory=list)]
+)
+class RegionalKeyPairCollection(object):
+    """
+    A :obj:`ReionalKeyPairCollection` is a collection of
+    :obj:`KeyPair` objects owned by a given tenant for a region.
+    """
+
+    def create_keypair(self, keypair):
+        """
+        Add a :obj:`KeyPair` to the list of keypairs
+        """
+        self.keypairs.append(keypair)
+        return self.keypair_by_name(keypair.name).key_json()
+
+    def keypair_by_name(self, name):
+        """
+        Return a :obj:`KeyPair` by name from the current keypairs list
+        """
+        for keypair in self.keypairs:
+            if keypair.name == name:
+                return keypair
+
+    def json_list(self):
+        """
+        JSON List of all :obj:`KeyPair` for a region
+        """
+        result = {"keypairs": []}
+        if len(self.keypairs) > 0:
+            keypairs_json = {}
+            for keypair in self.keypairs:
+                keypairs_json.update(keypair.key_json())
+            result = {
+                "keypairs": [
+                    keypairs_json
+                ]
+            }
+
```

Comment:
When I added the above code it started breaking. It would always return:
`{'keypairs': []}` even with keypairs in the collection.

Is it alright if I leave this loop as is? Reach is expecting a certain nested json structure from the api to be returned and I was able to achieve it with this loop.